<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSyncerClass</class>
 <widget class="QMainWindow" name="FileSyncerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileSyncer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_upload">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UPLOAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_OnlineFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_download">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWNLOAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_downloadall">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWNLOAD ALL(暂不可用)</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_transState">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>waitting for transport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LocalFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectLocal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_connectState">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DISCONNECTED</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileSyncer.qrc"/>
 </resources>
 <connections/>
</ui>
